docs/sdks/flutter/GETTING_STARTED.md

```diff
@@ -13,6 +13,15 @@ For **iOS** add your app name and Bundle ID, You can find your Bundle Identifier
 #### Android
 For **Android** add your app <u>name</u> and <u>package name</u>, Your package name is generally the applicationId in your app-level build.gradle file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.
 
+#### Linux
+For **Linux** add your app <u>name</u> and <u>package name</u>, Your package name is generally the `name` in your pubspec.yaml file. If you cannot find the correct package name, run the application in linux, and make any request with proper exception handling, you should get the application id needed to add in the received error message.
+
+#### Windows
+For **Windows** add your app <u>name</u> and <u>package name</u>, Your package name is generally the `name` in your pubspec.yaml file. If you cannot find the correct package name, run the application in windows, and make any request with proper exception handling, you should get the application id needed to add in the received error message.
```

Comment:
Let's add a link to our demo `pubspec.yaml` file, maybe from the playground?

Add the link with target="_blank" and rel="noopener"